lib/db/MiqSqlite/MiqSqlite3Table.rb

```diff
@@ -63,7 +63,7 @@ def decodeSchema
       return if @type != "table" || @name[0..6] == "sqlite_"  # Names beginning with sqlite_ are internal to engine
       @columns = []
       sql = @sql.gsub(/[\n\r]/, "")
-      re1 = /\s*CREATE\s+TABLE\s+(\w+)\s*\((.*)\)\s*/
+      re1 = /\s*CREATE\s+TABLE\s+'?(\w+)'?\s*\((.*)\)\s*/
```

Comment:
That's interesting - I've never seen quotes before.  Also note that technically the table name could have a `.` in it as well (i.e. `<schema_name>.<table_name>`)